src/client/styles/components/EBookList.scss

```diff
@@ -6,7 +6,8 @@ ul.nypl-ebooks-list {
   color: #0071bb;
   padding: 0;
   font-size: 1rem;
-  font-family: 'Kievit-Bold';
+  font-family: system-ui;
```

Comment:
Because you have this declared in `main.scss` as `$fontFamily` applied to the html element globally, you shouldn't need these additional `font-family` declarations. I'd pull these out, since that was the point of that abstraction anyway—that you can make global font-family changes relatively quickly.